### PR TITLE
Add function for moving student work from one class to another.

### PR DESCRIPTION
**Pull request by emcelroy**

### CODE REVIEW

**Comment by scytacki** [comment]
@emcelroy I think you mark this complete now and ask Noah or Doug for a review.

**Comment by emcelroy** [comment]
@knowuh Note that there's a corresponding draft PR for the portal at https://github.com/concord-consortium/rigse/pull/694

File: functions/src/api/helpers/portal-types.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  platform_user_id: string;
+  assignments: IPortalMoveStudentsAssignment[];
+}
+
+export interface IPortalMoveStudentsAssignment {
+  // change the first two to strings in the portal
+  old_resource_link_id: string;
+  new_resource_link_id: string;
+  tool_id: string;
+}
```
[the remark this answers is not in the document]
Comment (scytacki) [reply functions/src/api/helpers/portal-types.ts]: Good point @knowuh.  Additionally, @emcelroy I just realized that we should clarify in the `class_info_url` name above. I'm pretty sure it is the for the new class. So it should probably be named `new_class_info_url` to make that clear.

[the remark this answers is not in the document]
Comment (emcelroy) [reply functions/src/api/helpers/portal-types.ts]: @knowuh, @scytacki: Thanks.

Are these informative enough notes about the `context_id` values?

`// this should be the old class's class_hash value in the portal`
`// this should be the new class's class_hash value in the portal`

Would it be helpful to include notes about all the types? Or is it really only the `context_id` and `tool_id` that may need hints?

**Pull request merged**